README.md

```diff
@@ -28,12 +28,16 @@ Full documentation is available under [docs](https://strawberry-graphql.github.i
 from django.db import models
 
 class Fruit(models.Model):
+    """A tasty treat"""
     name = models.CharField(max_length=20)
     color = models.ForeignKey('Color', blank=True, null=True,
             related_name='fruits', on_delete=models.CASCADE)
 
 class Color(models.Model):
-    name = models.CharField(max_length=20)
+    name = models.CharField(
+      max_length=20,
+      help_text="field description",
```

Comment:
Maybe this was my suggestion, but it ended up with 2 spaces which is not usual for python =P

```suggestion
        max_length=20,
        help_text="field description",
```

Reply:
Ah yep, you're right, I don't know why I missed that inconsistency!